### PR TITLE
LCI 10 more

### PR DESCRIPTION
**Pull request by Simisays**
Credit to _boot_ on Discord for Tishana's Tidebinder

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Tishana's Tidebinder
+ManaCost:2 U
+Types:Creature Merfolk Wizard
+PT:3/2
+K:Flash
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounterAbility | TriggerDescription$ When CARDNAME enters the battlefield, counter up to one target activated or triggered ability. If an ability of an artifact, creature, or planeswalker is countered this way, that permanent loses all abilities for as long as Tishana's Tidebinder remains on the battlefield. (Mana abilities can't be targeted.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt:6]: CARDNAME

File: forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+SVar:TrigCounterAbility:DB$ Counter | TargetType$ Activated,Triggered | TgtPrompt$ Choose target ability | RememberCountered$ True | ValidTgts$ Card,Emblem | SubAbility$ DBEffect | SpellDescription$ Counter target activated or triggered ability. If a permanent's ability is countered this way, activated abilities of that permanent can't be activated this turn. (Mana abilities can't be targeted.)
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STLoseAB | RememberObjects$ Remembered | ConditionDefined$ Remembered | ConditionPresent$ Card.Artifact,Card.Creature,Card.Planeswalker | ConditionCompare$ EQ1 | Duration$ UntilHostLeavesPlay
+SVar:STLoseAB:Mode$ Continuous | Affected$ Card.IsRemembered | RemoveAllAbilities$ True | EffectZone$ Command | AffectedZone$ Battlefield | ValidCard$ Permanent.IsRemembered
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt:9]: Valid doesn't exist

File: forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Tishana's Tidebinder
+ManaCost:2 U
+Types:Creature Merfolk Wizard
+PT:3/2
+K:Flash
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Any | Destination$ Battlefield | Execute$ TrigCounterAbility | TriggerDescription$ When CARDNAME enters the battlefield, counter up to one target activated or triggered ability. If an ability of an artifact, creature, or planeswalker is countered this way, that permanent loses all abilities for as long as Tishana's Tidebinder remains on the battlefield. (Mana abilities can't be targeted.)
+SVar:TrigCounterAbility:DB$ Counter | TargetType$ Activated,Triggered | TgtPrompt$ Choose target ability | RememberCountered$ True | ValidTgts$ Card,Emblem | SubAbility$ DBEffect | SpellDescription$ Counter target activated or triggered ability. If a permanent's ability is countered this way, activated abilities of that permanent can't be activated this turn. (Mana abilities can't be targeted.)
+SVar:DBEffect:DB$ Effect | StaticAbilities$ STLoseAB | RememberObjects$ Remembered | ConditionDefined$ Remembered | ConditionPresent$ Card.Artifact,Card.Creature,Card.Planeswalker | ConditionCompare$ EQ1 | Duration$ UntilHostLeavesPlay
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt:8]: bad Condition

File: forge-gui/res/cardsfolder/upcoming/vito_fanatic_of_aclazotz.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Vito, Fanatic of Aclazotz
+ManaCost:2 W B
+Types:Legendary Creature Vampire Demon
+PT:4/4
+K:Flying
+T:Mode$ Sacrificed | ValidCard$ Permanent.Other | ValidPlayer$ You | Execute$ TrigGainLife | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice another permanent, you gain 2 life if this is the first time this ability has resolved this turn. If it's the second time, each opponent loses 2 life. If it's the third time, create a 4/3 white and black Vampire Demon creature token with flying.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vito_fanatic_of_aclazotz.txt:6]: what StackDescriptions you mean? it's a trigger...

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/vito_fanatic_of_aclazotz.txt:6]: You're right, it just that now it's not very clear which time it is

Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/vito_fanatic_of_aclazotz.txt:6]: Maybe a flag that adds times triggered to the Important Stack Objects (and therefore appends to end of Trigger Desc?)

Either way this is probably something to add to an issue for follow-up, nothing to hold up this PR over

File: forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt]: the Types were correct?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt]: Is it just the types+`inZoneBattlefield `?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/tishanas_tidebinder.txt]: yea sorry, should have been clearer

**Review by tool4ever — APPROVED** [review]

**Pull request merged**